api/apps/api/src/modules/scenarios/scenarios.service.ts

```diff
@@ -68,6 +68,7 @@ import {
   ScenarioCalibrationRepo,
 } from '../blm/values/scenario-calibration-repo';
 import { StartBlmCalibration } from './blm-calibration/start-blm-calibration.command';
+import { CancelBlmCalibration } from './blm-calibration/cancel-blm-calibration.command';
```

Comment:
[minor] I would recommend putting every public api of given module in root's index.ts of this module - to have a clear vision what can be used (i.e. avoid reaching into internal things). In future you may want to consider some rules around imports.